### DISCUSSION TITLE
Is Dast vs. Default templates vice versa only?

### DISCUSSION THREAD

**Discussion by iamthefrogy** [projectdiscovery/nuclei#5496] [Q&A]
Can't we use -dast along with default nuclei templates?
Should we have to use it separately only?

**Comment by ehsandeep** [comment]
@iamthefrogy Yes, as of now it's either way, as input for both kind of template is different, runing default templates with dast can produce unexpected matches.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
